test/functional/p2p_invalid_messages.py

```diff
@@ -82,7 +82,7 @@ def test_buffer(self):
         middle = self.nodes[0].getnettotals()['totalbytesrecv']
         assert_equal(middle, before + cut_pos)
         conn.send_raw_message(msg[cut_pos:])
-        conn.sync_with_ping(timeout=1)
+        conn.sync_with_ping(timeout=2)
```

Comment:
any ideas why it doesn't fail in bitcoin? They have 2 seconds here and it works.

Btw, let's move it to spare PR p2p_invalid_messages.py to get merged independently

Reply:
Spun off into [dash#6506](https://github.com/dashpay/dash/pull/6506) and merged, rebased on top of `develop`. Resolved in latest push!